summarizer/main.py

```diff
@@ -10,8 +11,16 @@ def train(hps):
     # For every split file
     for splits_file in hps.splits_files:
         print("Start training on {}".format(splits_file))
+
+        hps.current_dataset = splits_file
+        # Cosmetic for Tensorboard
+        if "splits/" in hps.current_dataset:
+            hps.current_dataset = hps.current_dataset.split("splits/")[1]
+        if "_splits" in hps.current_dataset:
+            hps.current_dataset = hps.current_dataset.split("_splits")[0]
+
```

Comment:
We may move this into utils/config.py directly as there is already code related to that